stack-8.10.yaml

```diff
@@ -4,7 +4,7 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-resolver: ghc-8.10.1
+resolver: lts-18.10
```

Comment:
You changed this to lts-18.6 in https://github.com/google/proto-lens/pull/417, and now it's lts-18.10. Is lts-18.10 correct?

Reply:
Yeah, again #417 was pending long enough that I forgot about it and move to a newer lts here.